cukas.js

```diff
@@ -106,6 +107,22 @@ class CukasGame {
             this.players[defencePlayerId].hand.push(this.deck.getRandomSet(Math.abs(this.players[attackPlayerId].hand.length - 6))); //make sure defence has 6 cards
         }
         //console.log(this.players);
+
+    }
+    compareCards(A,B){//1 - a>b -1 - a<b 0 - a=b
+      if(A.type==Card.TYPE_JOKER)return 1;
+      if(B.type==Card.TYPE_JOKER)return -1;
+      let a_trump=A.suit==trump;
+      let b_trump=B.suit==trump;
+      if(A.suit==B.suit || a_trump || b_trump){
+        let pow=A.rank;
+        if(a_trump)pow+=20;
+        pow-=B.rank;
+        if(b_trump)pow-=20;
+        return Math.sign(pow);
+      }
+      console.warn("Warning: these are not comparable");
+      return false;
```

Comment:
Probably should throw an error

Reply:
no since in the game someone could propose to put an invalid card
so unless there is an if statement that tests this it shouldn't throw errors